src/javascripts/ng-admin/Main/component/service/config/Field.js

```diff
@@ -19,6 +19,9 @@ define(function (require) {
         order: null,
         identifier: false,
         format: 'yyyy-MM-dd',
+        parse: function (date) {
+            return date;
```

Comment:
Shouldn't it create a `Date` object by default? That way, when serialized, it would be sent as ISO.